recipes/fancycompleter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fancycompleter" %}
+{% set version = "0.7" %}
+{% set sha256 = "5044bf91766dde3addfea81aaf36bacce89e1a872a10567f57e51f981c531eb7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pyreadline  # [win]
+    - pyrepl      # [not win and py2k]
+
+test:
+  imports:
+    - fancycompleter
+
+about:
+  home: https://bitbucket.org/antocuni/fancycompleter
+  license: BSD
```

Comment:
Many packages don't ship the file but have that info somewhere in the metadata or the docs. If not we need to contact upstream and request it.